src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -34,16 +34,12 @@ export const RangeSlider = forwardRef<HTMLInputElement, RangeSliderProps>(
 
     return (
       <>
-        <div data-testid="flowbite-range-slider" className={twMerge(theme.root.base, className)}>
-          <div className={theme.field.base}>
-            <input
-              ref={ref}
-              type="range"
-              className={twMerge(theme.field.input.base, theme.field.input.sizes[sizing])}
-              {...props}
-            />
-          </div>
-        </div>
+        <input
+          ref={ref}
+          type="range"
+          className={twMerge(theme.field.input.base, theme.field.input.sizes[sizing], className)}
```

Comment:
Since you're doing this, it would be helpful to consolidate the theme API here since we're making the component much simpler, as it should be. It seems like you could change the theme to literally just `base` and `sizes` keys. 

Remember to consult [the Flowbite vanilla docs on RangeSlider](https://flowbite.com/docs/forms/range/#range-slider-example) for the classes in the theme. I think we have a mismatch already, before your PR.

Reply:
@tulup-conner Good point, thanks!